docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -1357,6 +1852,25 @@ Addressing is implemented as follows:
 	    return Namespace + hexdigest(name)[:64]
     }
 
+{% elif language == 'Rust' %}
+
+.. code-block:: rust
+    :caption: XO addressing
```

Comment:
I would remove this caption. The other captions add information (file, struct name, etc.), but this one just repeats the intro sentence.